nmqtt.nim

```diff
@@ -25,6 +25,8 @@ type
     host: string
     port: Port
     sslOn: bool
+    sslCertFile: string
```

Comment:
The brokers definition is `sslCert` and `sslKey`. Can we align the type naming? I prefer your proposal with the `File` appended.

https://github.com/zevv/nmqtt/blob/de90ffe0d830e7fbd99f1f36b319a43eea697a29/nmqtt.nim#L135-L140

Reply:
I had removed the `File` suffix but i think maybe you meant to update the MqttBroker definition and add the `File` suffix there?